### PR TITLE
Added unverified bots default intent.

### PR DESCRIPTION
**Pull request by AA1999**
Same as `i_default_intents` but also has `i_message_content` intent.

### CODE REVIEW

File: include/dpp/intents.h [hunk 1]
```diff
@@ -78,7 +78,9 @@ enum intents {
 	/// Privileged intents requiring ID
 	i_privileged_intents		= dpp::i_guild_members | dpp::i_guild_presences | dpp::i_message_content,
 	/// Every single intent
-	i_all_intents			= dpp::i_default_intents | dpp::i_privileged_intents
+	i_all_intents			= dpp::i_default_intents | dpp::i_privileged_intents,
+        // Unverified bots default intents
```
Comment (Commandserver) [inline include/dpp/intents.h:82]: Put a triple slash there, otherwise it's just a normal comment

**Comment by Commandserver** [comment]
Does unverified bots have message content intent enabled by default? 🤔

**Comment by braindigitalis** [comment]
no, this only applies to unverified bots added before the change not new ones created today 

the message content is still a privileged intent and should not be made default by dpp

File: include/dpp/intents.h (hunk 1 above)
Comment (AA1999) [reply include/dpp/intents.h]: Done

**Comment by AA1999** [comment]
Any unverified bot that is below 100 server should have no problems using message intents tho, right?
Discord made it clear it's only a thing for verified bots anyways

**Comment by braindigitalis** [comment]
> Any unverified bot that is below 100 server should have no problems using message intents tho, right? Discord made it clear it's only a thing for verified bots anyways (or the bots at 100 servers)

yes but it is still a privileged intent and you still have to go into developer page and turn it on... we would be introducing issues later when the bot grows and they then require approval to use the intent...

**Comment by AA1999** [comment]
It is called `unverified bots intent` for a reason

**Comment by braindigitalis** [comment]
still shouldn't be defaulted imho, it becomes something else later and people will get used to having it enabled

**Comment by braindigitalis** [comment]
hmm this actually looks different to what I saw before.... I'll merge it later

**Comment by Axyte** [comment]
> still shouldn't be defaulted imho, it becomes something else later and people will get used to having it enabled 

Agreed with this statement here.

**Comment by Axyte** [comment]
It's really unnecessary and in-fact over-thought imo.

**Comment by AA1999** [comment]
It's not replacing the `default_intents`
It's just a handy alternative for people making bots that are private/never gonna reach 100 servers

**Comment by Axyte** [comment]
> It's not replacing the `default_intents` It's just a handy alternative for people making bots that are private/never gonna reach 100 servers

I'm aware it's not replacing that, but I still feel the same. Rest is up to @braindigitalis.

**Comment by Commandserver** [comment]
> It's really unnecessary and in-fact over-thought imo.

I agree tbh

**Comment by AA1999** [comment]
We already have a `all` intents that people might wrongly get used to.
We could maybe rename it better but I feel like it's not really a bad addition

**Comment by Commandserver** [comment]
> We already have a `all` intents that people might wrongly get used to.
> We could maybe rename it better but I feel like it's not really a bad addition

I would suggest `i_defaults_and_message_intents`

**Comment by es183923** [comment]
That is literally just as verbose as `i_default_intents | i_message_content`

**Comment by AA1999** [comment]
Idk just feels like having aliases is fine


> > We already have a `all` intents that people might wrongly get used to.
> > We could maybe rename it better but I feel like it's not really a bad addition
> 
> I would suggest `i_defaults_and_message_intents`

Seems fine to me, we'll let brain decide

**Comment by AA1999** [comment]
Maybe `i_private_bots_default_intents`?

**Pull request merged**